src/classes/SobjectTriggerHandler.cls

```diff
@@ -0,0 +1,165 @@
+/********************************************************************************************************
+* This file is part of the Nebula Framework project, released under the MIT License.                    *
+* See LICENSE file or go to https://github.com/jongpie/NebulaTriggerFramework for full license details. *
+********************************************************************************************************/
+public abstract class SobjectTriggerHandler implements ISobjectTriggerHandler {
```

Comment:
Missing or incorrect sharing behavior on `SobjectTriggerHandler`. Please update the sharing clause to `with sharing`.